### ISSUE TITLE
Question how commands are being handled

### ISSUE THREAD

**Issue by davidmuc** [Ape/samsungctl#68]
Hello @Ape ,
your tool works almost perfectly with my Samsung QE55Q7F. Thank you very much for the contribution - and the updates by @pedrinho ! That said, not all commands are working (such as KEY_TV to switch to TV mode; or KEY_HDMI2 to select HDMI 2 input). I would like to make that work and need to understand how your tool passes on commands from the command line onto the television.

Are the possible commands stored somewhere in your script (and only the commands "known" are passed on to the television) - or are **all** commands that are entered passed on to the television? 

In other words: if I would fire the following command:
`$ python -m samsungctl --host 192.168.178.66 --port 8001 --method websocket --name OSX NONSENSE`
...will "NONSENSE" be passed through to the TV?

If commands have to be stored somewhere in your code, where can I add them?
Thank you!

**Comment by Ape** [comment]
The TV will receive the command as a string exactly as given to samsungctl. The problem is that different models support different subsets of commands, and they often do different things with the same command. Many times the command strings are very counterintuitive. 

The README contains a list of common commands, but it is probably not complete. Submit a pull request if you find more commands.
